### PR TITLE
Stub out v5 alpha 2 blog post

### PR DESCRIPTION
**Pull request by mdo**
Will update more content today.

/cc @XhmikosR

### CODE REVIEW

File: src/_posts/2020-09-29-bootstrap-5-alpha-2.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+title: Bootstrap 5 Alpha 2
+video:
+date: 2020-09-29 11:00:00
+---
+
+We're back at it again with a brand new alpha release of Bootstrap 5! Our second alpha has brought some new and improved features, color contrast improvements, improved helpers and utilities, and some documentation design updates.
+
+Check out the latest in the docs at <https://v5.getbootstrap.com>. [Read the release notes]() for a full list of changes since Alpha 1. Our [migration guide](https://v5.getbootstrap.com/docs/5.0/migration) has also been updated with a new section for Alpha 2.
+
+## Updated docs nav
```
Comment (MartijnCuppens) [inline src/_posts/2020-09-29-bootstrap-5-alpha-2.md:12]: This change isn't backported to v4 (yet), so this might prehaps be a little confusing?

Comment (XhmikosR) [reply src/_posts/2020-09-29-bootstrap-5-alpha-2.md:12]: But this whole blog post refers to alpha 2 not v4 :)

File: src/_posts/2020-09-29-bootstrap-5-alpha-2.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: Bootstrap 5 Alpha 2
+video: Gs069dndIYk
+date: 2020-09-29 11:00:00
+---
+
+We're back at it again with a brand new alpha release of Bootstrap 5! Our second alpha has brought some new and improved features, color contrast improvements, improved helpers and utilities, and some documentation design updates.
+
+Check out the latest in the docs at <https://v5.getbootstrap.com>. [Read the release notes]() for a full list of changes since Alpha 1. Our [migration guide](https://v5.getbootstrap.com/docs/5.0/migration) has also been updated with a new section for Alpha 2.
```
Comment (mdo) [inline src/_posts/2020-09-29-bootstrap-5-alpha-2.md:10]: Add release link

File: src/_posts/2020-09-29-bootstrap-5-alpha-2.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Container horizontal `padding` has been updated to match the row gutters variables and values.
+- Checkboxes and radios downsized to `1em` from `1.25em` for improved font scaling support.
+- Improved colors with green and cyan getting new values, plus higher contrast ratios all around.
+- Improved toast styling, now without `overflow: hidden`.
+- Badge padding increased a smidge.
+- Revamped "responsive embed" helpers, now called ratio helpers. New class names across the board, more flexible styles, and a new CSS variable for custom dynamic and responsive ratios.
+- Screen reader classes are now "visually hidden" classes.
+- New `.border-width` utility.
+
+See all the changes in the [v5 Alpha 2 project board](https://github.com/twbs/bootstrap/projects/21) and be sure to [read the Migration guide]() for details on what's changed since Alpha 1.
```
Comment (mdo) [inline src/_posts/2020-09-29-bootstrap-5-alpha-2.md:69]: Link to migration guide

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**